scripts/orchestration/qoder_dispatch_bridge.py

```
@@ -1168,9 +1168,36 @@ def binding_slug(value: Any, *, field: str) -> str:
         reviewer_slug,
     ]
-    if len(assigned_roles) != len(set(assigned_roles)):
+    creative_context = payload.get("creative_pilot_context")
+    synthesis_coordinator_aliases = False
+    if (
+        isinstance(creative_context, dict)
+        and creative_context.get("schema_version") == "creative_pilot_context.v2"
+        and creative_context.get("phase") == "synthesis"
+    ):
+        try:
+            validated_creative_context = validate_task_pilot_context(creative_context)
```

Flake8 (7.3.0)

[error] local variable 'validated_creative_context' is assigned to but never used

(F841)